rfc-119-merge-mainstream-browse-topics-and-taxons.md

```diff
@@ -0,0 +1,169 @@
+# RFC 119: Merge mainstream browse, specialist topic into to taxon content items
+
+## Summary
+
+Mainstream browse, specialist topic and taxon content items are functionally equivalent, in that they represent a category that groups content items. We would like to maintain future consistency between our taxonomy, mainstream browse and specialist topic titles. We propose merging the attributes of mainstream browse and topic content items into taxons and deprecating use of mainstream browse and specialist topics in the Publishing API and Content Store.
+
+## Problem
+
+We currently maintain three seperate ways to categorise content items - mainstream browse pages, specialist topics and taxons. There is currently ongoing work to make sure mainstream browse and specialist topics have equivalents in the taxonomy i.e. mainstream browse and specialist topics become a subset of taxons. This is being done by Content Designers who are liaising with departments to come to agreement.
+
+Once this work is complete we want to maintain alignment and prevent divergence from taxons. We would have to put constraints in collections publisher and content tagger (or any future apps) to ensure that mainstream browse and specialist topics continue to be subsets of taxons. This would involve adding complex logic to the publishing system to deal with addition, changing or removal of these content items.
+
+## Proposal
+
+We propose merging the attributes of mainstream browse and specialist topic content items into **existing** taxon content items with the same `title`. (For clarity we aren't proposing the creation of new taxons or taxon tree.)
+
+Generally there are two main type of information we want to tranfer from mainstream browse and specialist topic to the equivalent taxon. The first is the links to content items. Currently content items (guidance pages, manuals etc..) do not contain links to mainstream browse or specialist topics. It's the reverse, mainstream browse pages and specialist topics have a list of content that's associated to them. This list we can migrate over to the taxon.
+
+Secondly, we'd carry across the relationships (links) between mainstream browse pages/specialist topics. This manifests as multiple hierarchies within the same taxons. This is because different relationships are useful for different things ie. the currently hierarchy for taxonomy is good for emails subscriptions and organising taxons, but not so great at modelling navigation. So we'd bring over the mainstream browse hierarchy to serve that purpose.
```

Comment:
I don't think there's any evidence that taxons are good for email subscriptions. We suspect the opposite: they are far too generic to represent any intent of the user, who will end up receiving all content tagged the taxon and its children, including things like "meeting minutes". The Notifications team are aiming to do a discovery that will include finding out whether we need to support taxon subscriptions at all.

I'm also not sure what we mean by "organising taxons" - in one sentence this reads as "the taxonomy is good for organising taxons".

Perhaps the conclusion is that it's not got for anything? 😆

Reply:
Ah yup - so here we are talking specifically about the **hierarchical** property of the taxonomy. I've removed the "organising taxons" as its confusing - was trying to describe that the hierarchy makes it easy to find specific taxons for publishers.

 I think the point to be made is that different hierarchical structures are used for different purposes. Would this be better?
 
```suggestion
Secondly, we'd carry across the relationships (links) between mainstream browse pages/specialist topics. This manifests as multiple hierarchies within the same taxons. This is because different relationships are useful for different things i.e. the current hierarchy for taxonomy is used for modelling emails subscriptions, but is less useful for modelling navigation. So we'd bring over the mainstream browse hierarchy to serve that purpose.
```

Reply:
That's clearer, thanks!

Minor: I'd suggest "used" instead of "useful" everywhere. This will hopefully become clearer with [my other comment](https://github.com/alphagov/govuk-rfcs/pull/119#discussion_r524295884) around the purpose of these three things.